pkg/auth/handler/webapp/authflowv2/forgot_password_otp.go

```diff
@@ -191,7 +193,11 @@ func (h *AuthflowV2ForgotPasswordOTPHandler) ServeHTTP(w http.ResponseWriter, r
 		}
 
 		result, err := h.Controller.AdvanceWithInput(r, s, screen, input, nil)
-		if err != nil {
+		if apierrors.IsKind(err, forgotpassword.PasswordResetFailed) {
```

Comment:
Instead of adding info here, I think we can add info in forgotpassword/service.go
In particular we add `otp_form: code` or `otp_form: link`.